playground/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: build
+build:
+	buildah bud -f Containerfile --no-cache -t playground .
```

Comment:
@lmilbaum is `make build` called somewhere? I don't see it

Reply:
It is not. I used it temporarily. Will remove it.